spec/features/users/user_visits_category_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "user can find things by category", type: :feature do
+  xit "can click on a category tag on the menu page" do
+    item = create(:item)
+    category = create(:category)
+    item.categories << category
+    item.save!
+
+    visit "/menu"
+    click_link_or_button("Specials")
+    assert page.current_path == menu_item_name_path
+  end
+
+  it "can see different category pages" do
+    item = create(:item)
+    category = create(:category)
+    item.categories << category
+    item.save!
+
+    visit "/menu/category/Specials"
+    expect(page).to have_content("Salted Caramel Peanut Butter Cup")
+  end
+end
```

Comment:
So I was thinking that we all need to decided on how to organize the `features` directory since there's not a whole lot of convention we've learned for that folder. I was thinking we could have `features/unauth-users/`, `features/auth-users/`, and `features/admins/`. I am open to other structures if you have a better idea on how our integration test folder should look.